docs/en/ingest-management/elastic-agent/install-elastic-agent-in-container.asciidoc

```diff
@@ -14,5 +14,7 @@ To learn how to run {agent}s in a containerized environment, see:
 
 * <<running-on-kubernetes-standalone>>
 
+* <<running-on-gke-standard-managed-by-fleet>>
```

Comment:
to simplify readability, we could make it looks like a sublist of `Run Elastic Agent on Kubernetes managed by Fleet`
<img width="586" alt="Screenshot 2022-11-14 at 17 41 22" src="https://user-images.githubusercontent.com/28299531/201716075-f4e06108-7fb5-4aaf-b310-112b977b5e8b.png">

```suggestion
** <<running-on-gke-standard-managed-by-fleet>>
```
wdyt?